addons/binding/org.openhab.binding.miele/ESH-INF/thing/dishwasher.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="miele"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Dishwasher -->
+	<thing-type id="dishwasher">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="xgw3000" />
+		</supported-bridge-type-refs>
+
+		<label>Dishwasher</label>
+		<description>This is a Miele@Home compatible dishwasher</description>
+
+		<channels>
+			<channel id="state" typeId="state" />
+			<channel id="program" typeId="program" />
+			<channel id="phase" typeId="phase" />
+			<channel id="start" typeId="start" />
+			<channel id="duration" typeId="duration" />
+			<channel id="elapsed" typeId="elapsed" />
+			<channel id="finish" typeId="finish" />
+			<channel id="door" typeId="door" />
+			<channel id="switch" typeId="switch" />
+		</channels>
+
+		<config-description>
+			<parameter name="uid" type="text">
+				<label>ID</label>
+				<description>The identifier identifies one certain appliance on the ZigBee network.</description>
+				<required>true</required>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+    <channel-type id="state" advanced="false">
+        <item-type>String</item-type>
+        <label>Status</label>
+        <description>Current status of the appliance</description>
+        <state readOnly="true"></state>
+    </channel-type>
+    
+	    <channel-type id="program" advanced="false">
+        <item-type>String</item-type>
+        <label>Program</label>
+        <description>Current program or function running on the appliance</description>
+        <state readOnly="true"></state>
```

Comment:
For the UI, it would be really cool to have the different options available here - this might mean that we need different channel types for the different appliances, though. Do you have any chance to find out the labels for the different options?